platform/servicecontrol.include.md

```diff
@@ -17,3 +25,9 @@ NOTE: ServiceControl _consumes_ messages that arrive in either the configured au
 By default ServiceControl stores information for 30 days, but it can be [customized](/servicecontrol/creating-config-file.md).
 
 Refer to the [Optimizing for use in different environments](/servicecontrol/servicecontrol-in-practice.md) article for more information about practical considerations.
+
+## ServiceMonitor
+
+To enable [ServiceMonitor](/servicecontrol) to collect metrics data [add the NServiceBus.Metrics](/nservicebus/operations/metrics.md) package to the solution endpoints and [configure them to send metrics data to ServiceMonitor](/servicecontrol/configure-nservicebusmetrics-for-servicemonitor.md).
+
+NOTE: All endpoints should be configured to send metrics to the same ServiceMonitor instance, unless the system has been sharded between multiple ServiceMonitor instances.
```

Comment:
I think we need to finish https://github.com/Particular/ServiceControl.Monitoring/pull/25 which is being driven by the Pulse UI before we can answer this definitively. I think it would be safe to say something like:

> The Monitoring Instance keeps data in memory for up to X minutes/hours. If the Monitoring Instance is restarted, this data is lost.